packages/web/src/lib/core/internal/common/LocalStorage.ts

```diff
@@ -0,0 +1,124 @@
+import { Storage } from "stitch-core";
+
+export default class LocalStorage implements Storage {
+  constructor(private readonly suiteName: string) {}
+
+  public get(key: string): any {
+    return localStorage.getItem(`${this.suiteName}.${key}`);
+  }
+  public set(key: string, value: string): any {
+    return localStorage.setItem(`${this.suiteName}.${key}`, value);
+  }
+  public remove(key: string): any {
+    return localStorage.removeItem(`${this.suiteName}.${key}`);
+  }
+}
+
+if (!window.localStorage) {
+  Object.defineProperty(
```

Comment:
[q] was this code you wrote? If not we should properly attribute it in a comment. or if you wrote the code then comment with a link to whatever specification you followed?

Reply:
I altered the code from Mozilla: https://developer.mozilla.org/en-US/docs/Web/API/Storage/LocalStorage. I'll add a comment.